utils/github/local.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+# `pip install …`
+import git # gitpython
+
+import functools
+import os
+import re
+
+
+class Local:
```

Comment:
What is local? Local copy or some local actions?

Reply:
It's a local repository interface.

Reply:
Ok, but maybe rename it?

Reply:
I added description to the class.